content/help/faq/security/does-istio-support-authorization.md

```diff
@@ -3,5 +3,5 @@ title: Does Istio Auth support authorization?
 weight: 110
 ---
 
-Yes. Starting from Istio 0.5 release, we provide Role Based Access Control for services in Istio mesh.
-[Learn more](/docs/concepts/security/#role-based-access-control-rbac).
+Yes. Starting from Istio 0.5 release, we provide Istio authorization for services in Istio mesh.
```

Comment:
Let's drop the "Starting from Istio 0.5 release" and just say "Istio provides authorization features for services in the mesh".

We should also change the title to just say "Does Istio support authorization controls?"